<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskStatus</class>
 <widget class="QWidget" name="DiskStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TabBar" name="widget_tab" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="TabDiskStatus" name="page_disk"/>
     <widget class="TabSmartStatus" name="page_smart"/>
     <widget class="TabDiskHealthDetection" name="page_health"/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabBar</class>
   <extends>QWidget</extends>
   <header>tabbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabDiskStatus</class>
   <extends>QWidget</extends>
   <header>TabDiskStatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabSmartStatus</class>
   <extends>QWidget</extends>
   <header>TabSmartStatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabDiskHealthDetection</class>
   <extends>QWidget</extends>
   <header>TabDiskHealthDetection.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
